scipy/stats/stats.py

```diff
@@ -201,7 +202,7 @@
            'f_oneway', 'F_onewayConstantInputWarning',
            'F_onewayBadInputSizesWarning',
            'PearsonRConstantInputWarning', 'PearsonRNearConstantInputWarning',
-           'pearsonr', 'fisher_exact', 'SpearmanRConstantInputWarning',
+           'pearsonr', 'fisher_exact', 'barnard_exact', 'SpearmanRConstantInputWarning',
            'spearmanr', 'pointbiserialr',
```

Comment:
Linter complains it's too long as well, this should fix it. 
```suggestion
           'pearsonr', 'fisher_exact', 'barnard_exact', 
           'SpearmanRConstantInputWarning', 'spearmanr', 'pointbiserialr',
```